README.md

```diff
@@ -56,7 +56,7 @@ If you don't want to use annotations but prefer raw YAML, use the following synt
 ```yaml
 foo:
     path:      /foo/{bar}
-    defaults:  { _controller: 'controller.Path\To\Your\Action' } # this is the name of the autoregistered service corresponding to this controller
+    defaults:  { _controller: 'Path\To\Your\Action:__invoke' } # this is the name of the autoregistered service corresponding to this controller
```

Comment:
I'm pretty sure this can just be `Path\To\Your\Action` - if it has `__invoke` on it (and this `id` lives in the container), it'll just work